example/rocketmq_v1alpha1_broker_cr.yaml

```diff
@@ -38,6 +38,8 @@ spec:
   size: 1
   # nameServers is the [ip:port] list of name service
   nameServers: ""
+  # rocketMQName is the rocketmq name, must equal to nameserver.spec.rocketMQName and topictransfer.spec.rocketMQName
```

Comment:
We can not ensure user set insistent name correctly, is there a better way to do this?

Reply:
I did this because we don't have a rocketmq resource as the parent resource of the broker and nameserver. Without a common parent controller, we can only specify the connection of the child resource in the spec. So I hope to add a rocketmq api.

Reply:
In that case should we add rocketmq higher level api before this PR?

Reply:
I think so